app/src/main/java/com/ably/tracking/demo/publisher/secrets/AndroidBase64Encoder.kt

```diff
@@ -0,0 +1,8 @@
+package com.ably.tracking.demo.publisher.secrets
+
+import android.util.Base64
+
+class AndroidBase64Encoder : Base64Encoder {
+    override fun encode(input: String) =
+        String(Base64.encode(input.toByteArray(), Base64.DEFAULT)).replace("\n", "")
```

Comment:
Why do you need to do the `replace()` at the end? :thinking:

Reply:
No idea why, but the Android encoder is adding a new line a the end :/
Added a comment about it 54bcf2ff51be05150ce2ff9c00eba20714a2f88c